src/components/dropdown/dropdown.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../styles/variables';
+.select-container {
+  display: inline-block;
+  position: relative;
+}
+
+.select {
+  appearance: none;
+  background-color: #f5f5f5;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  padding: 8px 20px 8px 10px;
+  font-size: 16px;
+  width: 200px;
+  cursor: pointer;
+}
+.select-sort-by {
+  appearance: none;
+  background-color: transparent;
+  border: 1px transparent;
+  border-radius: 4px;
+  padding:9px;
+  font-size: 1rem;
+  font-family: inherit;
+  width: 200px;
+  cursor: pointer;
+  color: $primary-color;
+}
+/* Dropdown arrow style */
+.select-arrow {
+  position: absolute;
+  top: 50%;
+  right: 10px;
+  transform: translateY(-50%);
+  width: 0;
+  height: 0;
+  border-left: 6px solid transparent;
+  border-right: 6px solid transparent;
+  border-top: 6px solid $icon-color;
+}
+.option {
+  padding: 8px 10px;
+  cursor: pointer;
+  background-color: $background-color;
+  min-width: 220px;
+  min-height: 100px;
+  color: $primary-color
+}
```

Comment:
Please Nest Parent -> Child styling

Reply:
Nested style according to html structure.